### PR TITLE
adding bind buttons to farm controllers for climbing

### PR DESCRIPTION
**Pull request by MariaKore**

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -43,4 +45,7 @@ public class Constants {
     public static final int LEFT_SHOOTER_CAN_ID = 20;
     public static final int RIGHT_SHOOTER_CAN_ID = 21;
     public static final int FEEDER_CAN_ID = 22;
+
+    public static final double ONSTAGE_RAISE_ELEVATOR = Units.inchesToMeters(30.0); //TODO: TUNE THIS LATER
+    public static final double ONSTAGE_LOWER_ELEVATOR = Units.inchesToMeters(10.0); //TODO: TUNE THIS LATER
```
Comment (prensing) [inline src/main/java/frc/robot/Constants.java:50]: These constants should be in the Elevator class.

**Review by prensing — CHANGES_REQUESTED** [review]

**Review by coachFitz0807 — COMMENTED** [review]
OVerall, should we also have a TestSetElevatorLength that reads the SmatDashboard and sets the elevator to that length?
And in Elevator.java, REACHER_OFFSET_TOLERANCE_METERS should be OFFSET_TOLERANCE_METERS

**Comment by MariaKore** [comment]
Hello, I have fixed my mistakes. Now some constants are in the Elevator class and in Elevator.java REACHER_OFFSET_TOLERANCE_METERS is now OFFSET_TOLERANCE_METERS. I am not sure what is SmatDashboard you mean SmartDashboard that displays the bot's data in real time and allow you to control bot variables? Is the TestSetElevatorLength is supposed to be a class or a file? We have SetElevatorLength which is a file in the commands folder.

**Comment by MariaKore** [comment]
Hello, I have fixed my mistakes. Now some constants are in the Elevator
class and in Elevator.java REACHER_OFFSET_TOLERANCE_METERS is now
OFFSET_TOLERANCE_METERS. I am not sure what is SmatDashboard you mean
SmartDashboard that displays the bot's data in real time and allow you to
control bot variables? Is the TestSetElevatorLength supposed to be a class
or a file? We have SetElevatorLength which is a file in the commands folder.

**Review by coachFitz0807 — COMMENTED** [review]
Don't worry about using the SmartDashboard in the TestSetElevatorCommand. That is being done int he addElevatorTest branch.
For the fixes tothe constants, did you commit those changes and push them? I am not seeeing them.

**Comment by MariaKore** [comment]
Oh I didn't commit them I'll do it as soon as possible.

**Review by coachFitz0807 — COMMENTED** [review]
After you commit, you also have to push so it goes up to Github and the rest of the team can see it. I'm not seeing your changes.

**Review by prensing — APPROVED** [review]

**Pull request merged**